pkgs/applications/audio/jackmix/default.nix

```diff
@@ -1,9 +1,11 @@
 { stdenv, fetchurl, pkgconfig, sconsPackages, qt4, lash, libjack2, jack ? libjack2, alsaLib }:
 
 stdenv.mkDerivation {
-  name = "jackmix-0.5.2";
+  pname = "jackmix";
+  version = "0.5.2";
+
   src = fetchurl {
```

Comment:
This should probably use fetchFromGitHub.

Reply:
It would change the final output. I am not trying to do this here and now.